app/helpers/application_helper.rb

```diff
@@ -1,6 +1,11 @@
-module ApplicationHelper
+module ApplicationHelper 
   def max_width
     # 後に条件分岐が必要
-    "mw-xl"
+    if devise_controller?
+      "mw-sm"
+    elsif controller.action_name == "index"
+      "mw-xl"
+    end
+
```

Comment:
詳細（`show`）ページもありますので，ひとまずこの形式にしておきましょうか。

```suggestion
    if devise_controller?
      "mw-sm"
    elsif controller.action_name == "index"
      "mw-xl"
    else
      "mw-md"
    end
```